src/queue.cpp

```diff
@@ -259,7 +259,6 @@ cl_int cvk_command_queue::wait_for_events(cl_uint num_events,
 }
 
 void cvk_executor_thread::set_queue(cvk_command_queue* queue) {
```

Comment:
I think we should remove this function. And get the profiling property from the queue of the first command in `cvk_executor_thread::executor`.

Reply:
I thought about it but decided to keep it as it provides a better separation of concerns, i.e. the executor pool doesn't need to know that executor threads need to be associated to queues because of profiling, they just have to know an association is needed.

Reply:
Note that #424 removes the need of `m_profiling` in the `cvk_executor_thread` (even so it seems I forget to remove it in the PR). It would leave `set_queue` completely empty.

Reply:
Alright, let's remove it under #424 then.